app/environments/brassbirmingham/brassbirmingham/envs/classes/road_location.py

```diff
@@ -21,7 +21,7 @@ class RoadLocation:
 
     def __init__(
         self,
-        networks: List[Town | TradePost],
+        networks: List[str],
```

Comment:
Why not `Town | TradePost`?

Reply:
When the `ROAD_LOCATION`s are initialized in `consts.py` the parameters are entered as strings. Also it makes more sense imo because it's less duplicated objects if that makes sense?